about/billing.html.markerb

```diff
@@ -42,22 +48,59 @@ Stopped Machines are billed based on their root file system (rootfs) usage per s
 
 For example, a Machine described in your dashboard as having 1GB of rootfs stopped for 10 days in the entire month will cost $0.05. If you have 3 stopped Machines of 1GB rootfs each stopped for 30 days it will cost $0.45 ($0.15 x 3).
 
+---
+
 ## GPU billing
 
 GPUs are billed per second that the attached Fly Machine is running (the time they spend in the `started` state), based on the per hour cost of the GPU card. See [pricing for GPUs](/docs/about/pricing/#gpus-and-fly-machines).
 
 You're also billed for the Fly Machine separately from the GPU.
 
+---
+
 ## Volume billing
 
 Volume billing is pro-rated to the hour and we subtract the free allowances first for the Launch, Scale, and Enterprise plans. For details, see [Volume pricing](/docs/about/pricing/#persistent-storage-volumes).
 
 If you create a volume, you will be charged for it. You’re billed for volumes that aren’t attached to Machines, and for volumes that are attached to Machines in any state, including stopped Machines.
 
+---
+
+## Unified Billing
+
+Unified billing allows you to consolidate billing for multiple organizations under a single organization. This means  You'll receive a single invoice for all organizations under unified billing. This feature is useful for managing billing for multiple organizations, such as when you have multiple teams or projects.
+
+**Billing Organization**: a normal organization with a payment method linked to it. This organization is responsible for paying for the resources used by all organizations linked to it.
+
+**Linked Organization**: an organization that has its billing linked to a another organization. It otherwise functions a normal organization
+
+### How Unified Billing Works
+
+- You'll receive a single invoice for all organizations under unified billing
+- Billing Organization must have a payment method on file (credit card or Stripe Link)
+- Linked Organizations inherit the plan of their Billing Organization
+- Linked Organizations share credits with their Billing Organization
+
+### How to use Unified Billing
+
```

Comment:
I think we should make this more "how to".

Something using what you have but more like:

Choose an organization to be the Billing Organization. Any organization on a plan can become the Billing Organization. (Question: does this include Pay As You Go? What about legacy plans?) You can use an existing organization if you have one or create one from the [new organization page](/organizations/new) in the dashboard.

Create a new organization using the [dashboard](/organizations/new), where you'll have the option to link it to a Billing Organization. This new org will be a Linked Organization. You can create and link multiple organizations to a single Billing Organization.

(not happy with the last paragraph above, but we can rework it)

Reply:
>  (Question: does this include Pay As You Go? What about legacy plans?)

any plan. I'd like for us to avoid talking about plans in at all with this feature, actually.

> Create a new Linked Organization using the [dashboard](https://github.com/organizations/new),

it's not actually from then dashboard. it's from the new organization page which isn't always obvious. that's why I was linking to that page specifically. 

---
@andie787 I've updated the copy. It's much simpler.

I decided lead with instructions on how to create a Linked Organization (since that is the "new" organization type that's being added by this feature).

Creating a billing organization is very obvious from the UI. The user won't have any organizations to select from in the dropdown if they don't have a billing organization (and there are instructions to get them to create one). If we start to get questions about "how do I create a billing organization" then I think we'll add it. I'm inclined to not confuse the user into thinking that there's a special thing called a "billing organization" when it's way more simple than that

Reply:
This all makes sense. And I like/agree with not mentioning plans at all.

I think I end up calling anything in the UI the "dashboard". (TBD: We should probably standardize how we talk about these pages and how we do links in docs!)